apps/snitch_core/test/data/model/property_test.exs

```diff
@@ -0,0 +1,70 @@
+defmodule Snitch.Data.Model.PropertyTest do
+  use ExUnit.Case
+  use Snitch.DataCase
+
+  import Snitch.Factory
+  alias Snitch.Data.Model.Property
+  alias Snitch.Data.Schema.Property, as: PropertySchema
+
+  @params %{
+    name: "Sapphire Radeon",
+    display_name: "processors"
+  }
+
+  describe "create/2" do
+    test "create sucessfully" do
```

Comment:
@Jeshan7 the way you are writing describe and test. It would read "create/2 create sucessfully". 

This doesn't read nicely. I would suggest you look at other test cases.

Reply:
Okay